lighthouse-core/lib/emulation.js

```diff
@@ -27,9 +27,30 @@ const NEXUS5X_EMULATION_METRICS = {
   },
 };
 
+/**
+ * Desktop metrics adapted from emulated_devices/module.json
+ * @type {LH.Crdp.Emulation.SetDeviceMetricsOverrideRequest}
+ */
+const DESKTOP_EMULATION_METRICS = {
+  mobile: false,
+  screenWidth: 1366,
+  screenHeight: 768,
```

Comment:
@paulirish it's not 100% clear to me what of these are necessary to zero-out any potential previous emulation 🤔